tests/serial/map-if-necessary/two-solvers/without-substeps/SerialExplicit.cpp

```diff
@@ -0,0 +1,33 @@
+#ifndef PRECICE_NO_MPI
+
+#include "../helper.hpp"
+#include "testing/Testing.hpp"
+
+#include <precice/precice.hpp>
+#include <vector>
+
+BOOST_AUTO_TEST_SUITE(Integration)
+BOOST_AUTO_TEST_SUITE(Serial)
+BOOST_AUTO_TEST_SUITE(MapIfNecessary)
+BOOST_AUTO_TEST_SUITE(TwoSolvers)
+BOOST_AUTO_TEST_SUITE(WithoutSubsteps)
+
+BOOST_AUTO_TEST_CASE(SerialExplicit)
+{
+  PRECICE_TEST("One"_on(1_rank), "Two"_on(1_rank));
+
+  // We receive the new start sample
+  std::vector<int> readMappings{1, 1, 1};
```

Comment:
Are the mappings here 'initial data (0), 1 and 2' or the the last time step (3) actually mapped. From what I understand the third timestep shouldn't be mapped.

Reply:
They are `{after first time window, after second time window, after third time window}` and are checked on the second participant of the serial scheme.

Starting with the mappings of the initial data would probably be very useful, especially for the given case of a `second` participant.

You are right though, there shouldn't be a mapping after the last time window.

Reply:
You have a good eye for details! First and second are swapped in the configuration.

I'll make them consistent.